docker/bootstrap_internal.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+export CUSTOMSETTINGS="docker.localsettings-docker"
```

Comment:
not quite following this, but I thought settings modules had to have importable python names (`localsettings-docker` doesn't seem to fit that criteria)